spec/amp-var-substitutions.md

```diff
@@ -102,6 +102,12 @@ Link substitution requires per-use opt-in as an added security measure and to af
 <a href="https://example.com?client_id=CLIENT_ID(bar)&abc=QUERY_PARAM(abc)" data-amp-replace="CLIENT_ID,QUERY_PARAM">Go to my site</a>
 ```
 
+Variable substitution works well with dynamic parameters added to link href through `data-amp-addparams`. Parameters provided as part of `data-amp-addparams` that requires variable substitution has to be specified on `data-amp-replace` like
```

Comment:
I'd like to suggest you:

1. Add a level 4 subheading ( `#### Appending parameters to the href`).
2. Rewrite the information as follows (of course, assuming I'm technically correct ;) ):

If you need to append dynamic parameters to the `href`, specify the parameters by using the `data-amp-addparams` attribute.  Any substitution parameters that you specify in `data-amp-addparams` must also be specified in `data-amp-replace`, as in the following example: 

...